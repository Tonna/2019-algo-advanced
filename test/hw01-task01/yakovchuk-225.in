30
1 1 2 1 1 2 6 2 1 9 2 1 8 11 4 7 6 16 9 9 3 13 14 22 4 18 1 9 25
7
22 6
5 6
20 10
11 5
8 2
18 9
25 5
