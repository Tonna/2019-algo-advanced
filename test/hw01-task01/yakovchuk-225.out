9
11
30
23
7
17
30
